core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -49,6 +49,24 @@
           <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'aboutMenu', {tab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content" href="https://oppiafoundation.org/volunteer" target="_blank" rel="noopener" translate="I18N_TOPNAV_GET_INVOLVED"></a></li>
         </ul>
       </li>
+      <li ng-if= "SHOW_CLASSROOM_CALLOUT" ng-class="{'open' : $ctrl.activeMenuName === 'classroomMenu'}" uib-dropdown ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_CLASSROOM" aria-hidden="false" class="nav-item  oppia-navbar-clickable-dropdown protractor-test-classroom-oppia-list-item">
```

Comment:
Are you sure this is working? 
I don't see how this file gets access to `SHOW_CLASSROOM_CALLOUT` boolean from constants.ts. You'll probably have to give it access via the top-navigation-bar.directive.ts file.
To check if this works, set the boolean to true and see if the changes you've made are visible.

Reply:
It wasn't working. I have updated the top-navigation-bar.directive.ts file and accessed the SHOW_CLASSROOM_CALLOUT flag. Toggling the flag values makes it work. PTAL! :)

Reply:
Just to confirm -- this is working now right?

Reply:
Can you confirm please?